pages/api/profile.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable camelcase */
```

Comment:
Is it only for the line `        Authorization: bearerToken,`? If yes, should we disble this eslint check on a single line only? :)

Reply:
actually, we don't need to disable anything at all, it came copied from `api/login.js` file 😝  Thanks for noticing!